### ISSUE TITLE
Create an Express APP

### ISSUE THREAD

**Issue by vishwac09** [covid19india/api#141]
File sizes are increasing daily, amount of data downloaded per request is also huge. To improve the APP's overall performance and response time, please introduce a proper REST powered Backend.

Express is great, can stream data. Let me know if i can help in this,

**Comment by vipulw2011** [comment]
@vishwac09 If this request is approved, I will also like to contribute to development. I also think proper api's to be made as data is increasing.

**Comment by vipulwairagade** [comment]
@vishwac09 If this request is approved, I will also like to contribute to development. I also think proper api's to be made as data is increasing.

**Comment by JunaidBabu** [comment]
How do you suggest we do this? Data is increasing day by day and we are still handling the data from Google Sheets. If we are moving to and Express App, we would have to ditch the sheets? Or we'll have to maintain a UI to add data to database?

**Comment by vishwac09** [comment]
We can start with creating Endpoints such as /states/stateID/counts, /state/stateID/districts, /states/stateID/patients, /rawData, /totalStats etc As of now we can start reading data from these same json files and send back to Front end. So technically our main source will be the same JSON files.
If in future we are planning to move the data into SQL, we can write a small script (JS/Bash) which reads from source files, iterates and inserts data in Database Schema no need to create UI to insert data. 
Let know your thoughts.

**Comment by rahulmathews** [comment]
I can also help with developing REST powered Express API. But my suggestion is instead of using relational database as SQL, we can use MongoDb which uses the same object schema as JSON. Let me know if I can help you.....

**Comment by rohitjmathew** [comment]
I can help with express API as well as MongoDB. Do reach out if you want help

**Comment by JunaidBabu** [comment]
We were able to make modifications in the Sheets data structure to handle the huge amount of data. For now this is holding well for the main [site](https://www.covid19indai.org).

Another issue with pivoting to a separate framework is that, there are continuous changes in the data formats we receive from the bulletins. This would require high maintenance on the Express or other frameworks.

Closing this issue for now. Will reopen when there is a need in the future.

Thank you
